memento/src/main/java/com/iluwatar/memento/Star.java

```diff
@@ -71,7 +71,7 @@ public void timePasses() {
 
   StarMemento getMemento() {
 
-    StarMementoInternal state = new StarMementoInternal();
+    var state = new StarMementoInternal();
```

Comment:
For Memento pattern we need to mirror the changes in the `README.md` explanation